Casamari/MNC008.xml

```diff
@@ -328,43 +327,47 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                              ʾIyasus</persName> is mentioned on <locus target="#16v"/>.</seg>
                      </handNote>
 
                      <handNote xml:id="h2" script="Ethiopic" corresp="#ms_i2">
                         <locus from="16v" to="25r"/>
                         <seg type="ink">Black, red</seg>
                         <date notBefore="1800" notAfter="1976">19th/20th century</date>
-                        <desc>Mediocre hand with small characters. The writing conveys an uneven
-                           picture, because the space between the ruled lines varies and the writing
-                           lines were not drawn straight (<locus target="#19r #24r"/>). <persName
-                              ref="PRS13280WaldaAma" role="scribe">Walda ʾAmānuʾel</persName> is
-                           mentioned as the scribe on <locus target="#25r"/>.</desc></handNote>
+                        <seg type="script">Mediocre hand with small characters. The writing conveys
+                           an uneven picture, because the space between the ruled lines varies and
+                           the writing lines were not drawn straight (<locus target="#19r #24r"/>).
+                              <persName ref="PRS13280WaldaAma" role="scribe">Walda
+                              ʾAmānuʾel</persName> is mentioned as the scribe on <locus
+                              target="#25r"/>.</seg>
+                     </handNote>
 
                      <handNote xml:id="h3" script="Ethiopic" corresp="#ms_i3 #ms_i4">
                         <locus from="25v" to="40r"/>
                         <seg type="ink">Black, red</seg>
                         <date notBefore="1800" notAfter="1976">19th/20th century</date>
-                        <desc>Rushed hand of a scribe named <persName ref="PRS13281HayMar"
-                              role="scribe">Ḫāyla Māryām</persName> (<locus target="#35r"/>). Since
-                           the handwriting becomes highly irregular with the beginning of the
-                           incomplete <ref target="#ms_i4"/> (from <locus target="#35v"/> onward,
-                           see esp. the change of appearance from <locus target="#38r"/> to <locus
-                              target="#39v"/>), it seems quite possible that <ref target="#ms_i4"/>
-                           was written by another, yet inexperienced scribe.</desc></handNote>
+                        <seg type="script">Rushed hand of a scribe named <persName
+                              ref="PRS13281HayMar" role="scribe">Ḫāyla Māryām</persName> (<locus
+                              target="#35r"/>). Since the handwriting becomes highly irregular with
+                           the beginning of the incomplete <ref target="#ms_i4"/> (from <locus
+                              target="#35v"/> onward, see esp. the change of appearance from <locus
```

Comment:
I think we should take note (also in our heads) of the abbreviations we are using, in order to convert, e.g., all "especially" into "esp." throughout the catalogue and indicate it in a table of abbreviations

Reply:
I noted it down.